{
  expectedException.expect(ComponentException.class);
  expectedException.expectCause(instanceOf(ResolverException.class));
  callAndThrowException(new ResolverException(CoreMessages.createStaticMessage("")),null);
}
